scripts/php-xhprof.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Require repo deps
+/vagrant/scripts/epel.sh
+
+# install php-xhprof and graphviz
+yum install -y php-xhprof graphviz --enablerepo=epel --enablerepo=webtatic
+
+# clone the SS xhprof repo as we need that
+if [ ! -d /vagrant/xhprof ]; then
+	git clone https://github.com/sminnee/xhprof /vagrant/xhprof
+fi
+
+# set up the config
+cp -f /vagrant/xhprof/xhprof_lib/config.sample.php /vagrant/xhprof/xhprof_lib/config.php
+
+# replace the settings that don't make sense
+sed -i 's/^\$controlIPs = array();/$controlIPs = false;/i' /vagrant/xhprof/xhprof_lib/config.php
+sed -i '/^\$controlIPs\[\]/d' /vagrant/xhprof/xhprof_lib/config.php
+sed -i 's/http:\/\/localhost\/xhprof\/xhprof_html/http:\/\/localhost:1090\/xhprof_html/i' /vagrant/xhprof/xhprof_lib/config.php
+
+# create the tmpxh_dot error file
+touch /tmpxh_dot.err
+chmod 0777 /tmpxh_dot.err
+
+# sym link xhprof repo - needed to access _ss_env file
+ln -s /vagrant/xhprof/ /var/www/xhprof
+
+# add a port listener to apache for xhprof access
+echo "Listen 1090
+
+<VirtualHost *:1090>
+DocumentRoot /var/www/xhprof/
+ServerName localhost
+</VirtualHost>" > /etc/httpd/conf.d/xhprof.conf
+
```

Comment:
Same as for the SQL code. I think this would be better of imported instead of hardcoded.